src/assets/data/starter-pack-info.js

```diff
@@ -0,0 +1,299 @@
+      },
+      {
+        name: "PeterPortal - An Unofficial API for UCI",
+        link: "https://api.peterportal.org/",
+        tooltip:
+          "PeterPortal API provides consolidated data about UCI professors, courses, and grades."
+      }
+    ]
+  },
+  {
+    name: "Gaming",
+    description: "Build the next chart topping indie game!",
+    packs: [
+      {
+        name: "Unity",
+        link: "https://docs.unity3d.com/Manual/index.html",
+        tooltip:
+          "Official documentation for Unity, a cross-platform game engine"
+      },
+      {
+        name: "Godot",
+        link: "https://godotengine.org/",
+        tooltip:
+          "Godot provides a huge set of common tools for game development"
+      },
+      {
+        name: "PyGame",
+        link: "https://www.pygame.org/wiki/GettingStarted",
+        tooltip:
+          "Official documentation for PyGame, a cross-platform set of Python modules designed for writing video games"
+      }
+    ]
+  },
+  {
+    name: "Tools",
+    description: "Gotta build your projects with something!",
+    packs: [
+      {
+        name: "Git",
+        link: "https://github.com/zpinto/learn-git-with-hack",
+        tooltip:
+          "A complete tutorial on how to get started with Git, a version-control system"
+      },
+      {
+        name: "GCP Credits",
+        link:
+          "https://docs.google.com/forms/d/e/1FAIpQLScSPqesC4HwHu1HmWqgI3HD_-N0syDsJBqVM_El4S-fz7IRLw/viewform",
+        tooltip: "Access to free credits for hosting your project on GCP!"
+      },
+      {
+        name: "Ngrok",
+        link: "https://ngrok.io",
+        tooltip:
+          "Instantly create a public HTTPS url for a website running locally on your development machine using Ngrok"
+      },
+      {
+        name: "Postman",
+        link: "https://www.getpostman.com/",
+        tooltip:
+          "A great tool for testing RESTful APIs made by others or yourself"
+      },
+      {
+        name: "Replit",
+        link: "/starter-packs/replit",
+        tooltip:
+          "Collaborative in-browser IDE that makes it easy to get hacking!",
+        content: (
+          <>
+            <h2>Replit</h2>
+            <p>
+              Replit is a collaborative in-browser IDE that requires zero
+              additional setup and supports over 50 programming languages, such
+              as Python, C++, HTML, and CSS. Similar to Google Docs, you can
+              collaborate with others on code in real time. Replit is providing
+              all HackUCI 2021 attendees with 3 months free access to their
+              “Hacker” plan, which includes private projects, 10x the storage,
+              4x the memory, 4x the speed, and more. Sign up for your account{" "}
+              <a href="https://repl.it/login?goto=/claim?code=HackUCI2021">
+                here
+              </a>
+              !
+            </p>
+          </>
+        )
```

Comment:
Remove
```suggestion
```

Reply:
Yepp and also the live tmr by Mara Hart!